src/tmx.c

```diff
@@ -162,7 +171,40 @@ tmx_tileset* tmx_get_tile(tmx_map *map, unsigned int gid, unsigned int *x, unsig
 		}
 		ts = ts->next;
 	}
-	
+
 	return NULL;
 }
 
+tmx_tile_prop* tmx_get_tile_props(tmx_map *map, unsigned int gid) {

```

Comment:
I was hesitant to make this change as it would break backward compatibility, but I agree that it should happen.